zipline/data/minute_bars.py

```diff
@@ -528,11 +528,16 @@ def write_cols(self, sid, dts, cols):
                                  dts.astype('datetime64[ns]'))
 
         ohlc_ratio = self._ohlc_ratio
-        open_col[dt_ixs] = (cols['open'] * ohlc_ratio).astype(np.uint32)
-        high_col[dt_ixs] = (cols['high'] * ohlc_ratio).astype(np.uint32)
-        low_col[dt_ixs] = (cols['low'] * ohlc_ratio).astype(np.uint32)
-        close_col[dt_ixs] = (cols['close'] * ohlc_ratio).astype(
-            np.uint32)
+
+        def convert_col(col):
+            """Adapt float column into a uint32 column.
+            """
+            return (np.nan_to_num(col) * ohlc_ratio).astype(np.uint32)
```

Comment:
I had run into the same bug as Eddie and we both had a fix. This just pulls the logic into a function and shares it for all the columns.